src/utils/symbol_utils.ml

```diff
@@ -169,6 +222,39 @@ let call_finder : (tid * tid) list Term.visitor = object
 end
 
 
+let get_calls (program : program term) : (tid * tid) list =
+  match !call_finder_run with
+  | true -> !found_calls
+  | false -> begin
+      call_finder_run := true;
+      found_calls := call_finder#run program [];
+      !found_calls
+  end
+
+
+let check_if_symbols_resolved (project : Project.t) (program : program term) (tid_map : word Tid.Map.t) : bool =
+  let extern = build_and_return_extern_symbols project program tid_map in
+  let extern = List.filter extern ~f:(fun e ->
+    match Stdlib.List.mem e.name extern_symbol_blacklist with
+    | false -> true
+    | true -> false
```

Comment:
Ocaml has a `not` operator for boolean values, you don't need to match them to invert them.